src/api.js

```diff
@@ -0,0 +1,28 @@
+var url = 'https://en.wikipedia.org/w/api.php';
+
+var params = {
+  action: 'query',
+  generator: 'geosearch',
+  prop: 'coordinates|pageimages',
+  ggscoord: '37.7891838|-122.4033522',
+  format: 'json',
+};
+
+url = url + '?origin=*';
+Object.keys(params).forEach(function (key) {
+  url += '&' + key + '=' + params[key];
+});
+
+fetch(url)
+  .then(function (response) {
+    return response.json();
+  })
+  .then(function (response) {
+    var pages = response.query.pages;
+    for (var page in pages) {
```

Comment:
Here's a case where `var` is actually an error. Well, it doesn't _throw_ an error, but that's JavaScript being nice to you. `var` doesn't allow for declaration in lexical scopes. That's pretty jargon-y, but what's happening here under the hood is that in the interpretation phase, a variable `page` is being created at the top of the function scope, then being assigned to in the execution phase.

That's probably _not_ what you thought was happening, which I expect is more like what `let` does: creating a variable `page` that is only available inside the `for` loop.

TL;dr, change this `var` to `let` 😅